src/styles/_variables.scss

```diff
@@ -0,0 +1,50 @@
+$displayLarge: 1250px;
```

Comment:
you have too many variables, as if your project have tons of pages

Reply:
I think it make my layout much flexible

Reply:
it's not just for this variable, comment is for the whole file

Reply:
and I told about flexibility of all my layout